src/app/about-us/page.tsx

```diff
@@ -1,22 +1,27 @@
 import React, { FC } from "react";
 import AboutHero from "@/components/about-us/AboutHero";
-import AboutOurMission from "@/components/about-us/AboutOurMission";
-import BoardMembers from "@/components/about-us/BoardMembers";
-import MissionStatement from "@/components/about-us/MissionStatement";
-import Timeline from "@/components/about-us/Timeline";
+import AboutOurMissionVision from "@/components/about-us/AboutOurMissionVision";
 import missionStatementData from "@/data/about-us";
-import PageHead from "@/components/about-us/PageHeader";
+import { Separator } from "@radix-ui/themes";
+import TimelineItem from "@/components/about-us/Timeline";
 
 const AboutUsPage: FC = () => {
-  const { missionStatement, aboutOurMission, timeline } = missionStatementData;
+  const { aboutOurMission, aboutOurVision, timeline } = missionStatementData;
 
   return (
-    <main className="">
+    <main>
       <AboutHero />
-      <MissionStatement missionStatement={missionStatement} />
-      <BoardMembers />
-      <AboutOurMission content={aboutOurMission} />
-      <Timeline items={timeline} />
+      <div className="flex mt-10 md:mt-20 bg-gray-50 h-full ">
+        <div className="flex flex-col py-[47px] lg:flex-row justify-center items-center gap-0 lg:gap-10 xl:gap-20 px-[50px] md:px-[117px] h-full">
+          <AboutOurMissionVision title="Mission" content={aboutOurMission} />
+          <Separator
+            orientation="vertical"
+            className="hidden lg:block h-[500px]"
```

Comment:
**suggestion (non-blocking):** Would be good to use RadixUI's breakpoint system if possible

**suggestion (blocking):** override the flex centering so that the Separator stretches to fill the vertical space, instead of specifying a static height.